### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Chadys**
I wrote the French translation file and I also moved out the `<p/>` tag that was erroneously inside a `{% blocktrans %}` in `confirmation.html` (I also removed the now superfluous translation in all other translation files).

### CODE REVIEW

**Comment by Chadys** [comment]
I see #379 has been merged, I will update my branch to remove superfluous translation also in russian language

**Comment by dekoza** [comment]
Thank you, @Chadys I'll merge this PR as soon as I get confirmation that this translation is OK as I can't do this myself.

File: djoser/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+#: templates/email/activation.html:10 templates/email/activation.html:22
+msgid "Please go to the following page to activate account:"
+msgstr "Veuillez cliquer sur le lien suivant pour activer votre compte :"
+
+#: templates/email/activation.html:13 templates/email/activation.html:25
+#: templates/email/confirmation.html:10 templates/email/confirmation.html:18
+#: templates/email/password_reset.html:14
+#: templates/email/password_reset.html:26
+msgid "Thanks for using our site!"
+msgstr "Merci d'avoir utilisé notre site !"
```
Comment (dekoza) [inline djoser/locale/fr/LC_MESSAGES/django.po]: Is the space before exclamation mark normal in French?

Comment (Chadys) [reply djoser/locale/fr/LC_MESSAGES/django.po]: Yes, in French most punctuation needs a space before

**Pull request merged**